card/src/test/java/com/adyen/checkout/card/StoredCardDelegateTest.kt

```diff
@@ -297,7 +297,8 @@ internal class StoredCardDelegateTest {
             observerRepository = PaymentObserverRepository(),
             storedPaymentMethod = storedPaymentMethod,
             publicKeyRepository = publicKeyRepository,
-            configuration = configuration,
+            configuration  =configuration,
```

Comment:
Oh no

Reply:
How did that not cause a gradle check fail? 🤔

Reply:
Did we disable ktlint for tests?

Reply:
Yeah some rules it seems